mypy.ini

```diff
@@ -57,6 +57,7 @@ files = src/sentry/analytics/,
         src/sentry/grouping/strategies/template.py,
         src/sentry/grouping/strategies/utils.py,
         src/sentry/incidents/charts.py,
+        src/sentry/ingest/billing_metrics_consumer.py,
```

Comment:
Should we move this to `src/sentry/sentry_metrics`, which is where the indexer consumer lives?

Reply:
Decided with @iker-barriocanal to leave it here for now.